### PR TITLE
Networks documentation updates

### PR DESCRIPTION
**Pull request by jasdeepg**

### CODE REVIEW

**Comment by jasdeepg** [comment]
@wleklinski can you push when you get a chance? feel free to review, but it's just updates to documentation with some additional assets.

File: FAQ/what-is-compass.html.md [hunk 1]
```diff
@@ -14,8 +14,8 @@ Compass provides the following functionality:
 
 * __Content Ingestion:__ If you have content that you’ve produced previously that you’d like made available to specific partners or other brands in our network, we can ingest your content archives into the platform to create a structured and engaging content library for your key partners. 
 
-* __Content Analytics:__ Compass tracks visitor engagement on all content published through the platform.
+* ___Content Sharing___: You can directly share content with your partners seamlessly.
```
Comment (wleklinski) [inline FAQ/what-is-compass.html.md]: `___` -> `__`

File: Discovering/commissioned-content.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
+---
+title: Commissioned content
+layout: article
+sort: 3
+Description: Learn how to commission content within Compass, and find it once it's produced.
```
Comment (wleklinski) [inline Discovering/commissioned-content.md]: Dummy quotes are dangerous in frontmatter so it's best to use typographer's quotes (`'` -> `’`), or quote the whole line.

File: FAQ/what-is-owned-content.md [hunk 3]
```diff
@@ -0,0 +1,11 @@
+---
+title: What is owned content?
+layout: article
+sort: 10
```
Comment (wleklinski) [inline FAQ/what-is-owned-content.md]: `description` is missing for several pages.

File: Measuring/view-analytics.md [hunk 4]
```diff
@@ -0,0 +1,43 @@
+---
+title: Viewing Analytics
```
Comment (wleklinski) [inline Measuring/view-analytics.md]: Casing is inconsistent for `title`s.

File: Publishing/publishing-to-multiple-places.html.md [hunk 5]
```diff
@@ -16,7 +16,7 @@ In this article, you’ll learn how to easily publish one piece of content in ma
 * Choose which piece of content you want to activate
 * Make sure you’ve properly connected your blog to Compass
 * If you are using the API, create a new connection in Settings (make sure you note your secret, key, and token)
-* You can read about Channels here.
+* You can read about Channels [here](http://docs.rootsrated.com/FAQ/what-is-a-compass-channel/).
```
Comment (wleklinski) [inline Publishing/publishing-to-multiple-places.html.md]: Likely best to make this a relative URL, e.g. `/FAQ/what-is-a-compass-channel/`.

File: Measuring/view-analytics.md [hunk 6]
```diff
@@ -0,0 +1,43 @@
...
+
+![Analytics KPIs](/images/analytics_kpis.png)
+
+Once you've published a few pieces and have started to see traffic, you can view your content analytics in the [Analytics tab](https://compass.rootsrated).com/analytics).
+
+In this article you will learn how to do the following:
+1. View your content analytics
+2. Change the date range to view a larger window
+
+## Before your start
```
Comment (wleklinski) [inline Measuring/view-analytics.md]: `your` -> `you`

File: Networks/share-content.html.md [hunk 7]
```diff
@@ -0,0 +1,26 @@
...
+Description: Learn how to share content with connections.
+---
+
+# Sharing content
+
+In this article, you’ll learn how to share content with network partners.
+
+## Before your start
+
+* [Set up](/Measuring/add-new-connection) your network
```
Comment (tyrp) [inline Networks/share-content.html.md]: This link 404s. Should be `/Networks/` probably

File: Measuring/view-analytics.html.md [hunk 8]
```diff
@@ -0,0 +1,40 @@
...
+Compass generates a unique tracking snippet for each article distributed from the platform. An analytics snippet is generated for each piece of content published from Compass. We display the following metrics within Compass:
+
+* Readers - the number visitors that spend more than 15 seconds on an article.
+* Average Reader Time - the average time a reader spends reading an article.
+* Average Scroll Depth - the depth a reader scrolls down the page. One thing to keep in mind is that scroll depth can be affected if the reader is on mobile versus desktop.
+* Engagement Rate - we consider a “bounce” anyone who leaves the page within 15 seconds of arriving.
+
+![Analytics KPIs](/images/analytics_kpis.png)
+
+Once you've published a few pieces and have started to see traffic, you can view your content analytics in the [Analytics tab](https://compass.rootsrated).com/analytics).
```
Comment (tyrp) [inline Measuring/view-analytics.html.md]: Extra `)` in the link formatting

File: Measuring/view-analytics.html.md [hunk 9]
```diff
@@ -0,0 +1,40 @@
...
+2. Change the date range to view a larger window
+
+## Before you start
+
+* Publish a few pieces of content to your blog
+* Drive traffic to that content over the course of a few days
+
+## View your analytics
+
+1. Simply navigate to the [Analytics tab](https://compass.rootsrated)
```
Comment (tyrp) [inline Measuring/view-analytics.html.md]: Link is incorrect

**Pull request merged**